tomcat/src/test/java/org/apache/catalina/connector/ConnectorTest.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.catalina.connector;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import support.TestHttpUtils;
+
+class ConnectorTest {
+
+    @Test
+    @DisplayName("켜져있는 서버 메인 페이지를 600개 요청 해본다.")
+    void request600() throws InterruptedException {
+        // given
+        int threadCount = 600;
+        ExecutorService pool = Executors.newFixedThreadPool(threadCount);
+        List<Future<?>> futures = new ArrayList<>();
+
+        // when
+        for (int i = 0; i < threadCount; i++) {
+            futures.add(pool.submit(() -> TestHttpUtils.send("/index.html")));
+        }
+        pool.awaitTermination(10, TimeUnit.SECONDS);
+
+        long doneCount = futures.stream()
+                .filter(Future::isDone)
+                .count();
+
+        // then
+        System.out.println("doneCount = " + doneCount);
```

Comment:
```suggestion
        // then
        long doneCount = futures.stream()
                .filter(Future::isDone)
                .count();
        assertThat(doneCount).isEqualTo(threadCount);
```
아무래도 콘솔에 출력하는 게 친근하죠. ㅎ ㅎ 
그러나 테스트 코드니까 이렇게 검증문을 넣을 수도 있지 않을까요?